lib/device.js

```diff
@@ -15,12 +15,14 @@ var State      = require('./state'),
     Shutdown   = require('./device/shutdown'),
     Audio      = require('./device/audio'),
     Radiodan   = require('radiodan-client'),
+    Promise    = Radiodan.utils.promise,
```

Comment:
'Promise' is defined in a future version of JavaScript. Use a different variable name to avoid migration issues.